topdown/json.go

```diff
@@ -203,6 +203,11 @@ func pathsToObject(paths []ast.Ref) ast.Object {
 		node := root
 		var done bool
 
+		// If it's a null JSON path, skip all further processing.
```

Comment:
nit, but it's an empty string, not null.. right?

Reply:
When I wrote that, I was thinking about how the empty string in the list of paths-to-be-filtered is translated into a blank `ast.Ref`, which is basically a null/nil value. I'll embellish the comment with more details.